### PR TITLE
CLC-6711 Bring shared Master up-to-date with ETS Junction deployment

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-6711

The changes are all currently running on the production Junction cluster via the `jqa` branch. This PR realigns our shared `master` branch before ETS moves to the newer JRuby version.

Although it's always nice to hear from the SIS crew, I'd guess that @johncrossman is the most likely reviewer at this point.

### CODE REVIEW

**Comment by raydavis** [comment]
The initial Travis fail was due to our old pal, https://jira.berkeley.edu/browse/SISRP-14570
Second try succeeded.

File: app/models/canvas/course_copy_import.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    def initialize(options = {})
+      super(options)
+      @canvas_course_id = options[:canvas_course_id]
+    end
+
+    def import_course_template(template_id)
+      wrapped_post request_path, {
+        'migration_type' => 'course_copy_importer',
+        'settings[source_course_id]' => template_id
```
Comment (johncrossman) [inline app/models/canvas/course_copy_import.rb]: @raydavis we don't need `#{...}` in hash key above?

Comment (raydavis) [reply app/models/canvas/course_copy_import.rb]: Nope, @johncrossman , that literally is the query string we send the Canvas API: https://canvas.instructure.com/doc/api/content_migrations.html

**Comment by johncrossman** [comment]
Thank you much, @raydavis

**Pull request merged**